packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -434,11 +434,6 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
       return false;
     }
 
-    if (resultCode != Activity.RESULT_OK) {
```

Comment:
Hmmm, https://developers.google.com/identity/sign-in/android/sign-in#start_the_sign-in_flow made it look like it's safe to assume that the `GoogleSignInResult` is non-null.  Are there conflicting docs (it wouldn't surprise me :-))